tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -55,7 +55,7 @@
     "IsCreateCalendarEnabled": true,
     "IsMajorVersionEnable": true,
     "IsMinorVersionEnable": false,
-    "IsForceCheckOut": true,
+    "IsForceCheckOut": false,
```

Comment:
Changed IsForceCheckOut  = false so that when documents gets added to doc library, those documents wont get checkedout automatically